projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -773,6 +773,8 @@ public static function enqueue_block_editor_assets() {
 			}
 		}
 
+		$initial_state['social']['featureFlags'] = $social_initial_state['featureFlags'];
```

Comment:
👋🏻 @gmjuhasz I'm working on unrelated, exploratory work (pbArwn-6yv-p2) and encountered an error originating from this addition. 

```
Warning:  Undefined variable $social_initial_state in /var/www/html/wp-content/plugins/jetpack/class.jetpack-gutenberg.phpb on line 776
```

I wondered if this line should be relocated to the conditional above it where `$social_initial_state` is defined, to avoid referencing the undefined variable. WDYT? 


```diff
diff --git a/projects/plugins/jetpack/class.jetpack-gutenberg.php b/projects/plugins/jetpack/class.jetpack-gutenberg.php
index 4e2d0af1bf..737a621601 100644
--- a/projects/plugins/jetpack/class.jetpack-gutenberg.php
+++ b/projects/plugins/jetpack/class.jetpack-gutenberg.php
@@ -772,9 +772,9 @@ class Jetpack_Gutenberg {
 
 				$initial_state['social']['connectionRefreshPath'] = $social_initial_state['connectionRefreshPath'];
 			}
-		}
 
-		$initial_state['social']['featureFlags'] = $social_initial_state['featureFlags'];
+			$initial_state['social']['featureFlags'] = $social_initial_state['featureFlags'];
+		}
 
 		wp_localize_script(
 			'jetpack-blocks-editor',

```

Reply:
Hey! Thanks for noticing that, I'll prepare a PR for that

Reply:
Created a fix: https://github.com/Automattic/jetpack/pull/38781